docs/docs/installation/docker/index.md

```diff
@@ -19,49 +19,17 @@ You should have permission to run Docker containers on the system.
 ### Software
 You'll need to install **[Docker](https://docs.docker.com/desktop/)** v23+
 
-### Databases
-
-#### Database for Mathesar's internal usage
-You'll need to:
-
-- Create or have a PostgreSQL user for Mathesar to use.
-- Create a PostgreSQL database for Mathesar's internal usage owned by that database user. (See the PostgreSQL [docs](https://www.postgresql.org/docs/13/ddl-priv.html) for more information.)
-    
-    Alternatively, you can make the user a `SUPERUSER` which will give the user access to all the databases. (See the see PostgreSQL [docs](https://www.postgresql.org/docs/13/sql-createrole.html) for more information.)
-
-- Ensure that this database can accept network connections from the machine you're installing Mathesar on.
-- Have the following information for this database handy before installation:
-    - Database hostname
-    - Database port
-    - Database name
-    - Database username
-    - Database password
-
-#### Databases connected to Mathesar's UI
-Have the following information for all databases you'd like to connect to Mathesar's UI before installation:
-
-- Database hostname
-- Database port
-- Database name
-- Database username (should be `SUPERUSER` or `OWNER` of the database, see above)
-- Database password
-
-!!! info "Databases are automatically created"
-    You don't need to create these databases before installation. Whenever the Docker container is started, Mathesar will attempt to create any specified databases that don't already exist, so long as the user has [`CREATEDB` privilege](https://www.postgresql.org/docs/13/sql-createrole.html).
-
 ## Installation Steps
 
 1. Run the Mathesar Docker Image
 
     ```bash
     docker run \
-      --detach
-      -e DJANGO_DATABASE_URL='<replace with a postgres connection string>' \
-      -e MATHESAR_DATABASES='(<unique_db_key>|<replace with a postgres connection array>)' \
-      -e SECRET_KEY='<replace with a 50 character string>' \
-      -e ALLOWED_HOSTS='.localhost, 127.0.0.1, [::1]' \
+      --detach \
       -v static:/code/static \
       -v media:/code/media \
+      -v postgresql_config:/etc/postgresql/ \
+      -v postgresql_data:/var/lib/postgresql/data \
```

Comment:
Is this correct? Am I missing something? I think this is misaligned with the volume specified in the dockerfile.